wics-site/Gemfile.lock

```diff
@@ -166,8 +168,17 @@ GEM
     minitest (5.10.3)
     multipart-post (2.0.0)
     net-dns (0.8.0)
+<<<<<<< HEAD:wics-site/Gemfile.lock
```

Comment:
These merge conflict markers definitely should not be in here (or preferably this entire file...I thought this file was added to `.gitignore`, I wonder what happened?)

Reply:
I'm not sure either! Thanks for pointing this out; totally missed the conflicts